cpp/cmake/thirdparty/get_ranger.cmake

```diff
@@ -26,8 +26,6 @@ function(find_and_configure_ranger)
     rapids_cpm_find(
       ranger 00.01.00
       GLOBAL_TARGETS "${global_targets}"
-      BUILD_EXPORT_SET cuspatial-exports
-      INSTALL_EXPORT_SET cuspatial-exports
```

Comment:
Without this change, `cuspatial` wheel builds fail like this:

```text
By not providing "Findranger.cmake" in CMAKE_MODULE_PATH this project has
asked CMake to find a package configuration file provided by "ranger", but
CMake did not find one.

Could not find a package configuration file provided by "ranger" with any
of the following names:

  rangerConfig.cmake
  ranger-config.cmake
```

`ranger` is a PRIVATE dependency of `libcuspatial`, so I don't think it needs to be included in this export set.

https://github.com/rapidsai/cuspatial/blob/1544e7bd16f11457436d1fb6a43da0db3aa9af57/cpp/CMakeLists.txt#L207